auth/auth.go

```diff
@@ -183,34 +187,21 @@ func SearchAccount(query string) []gin.Account {
 	return results
 }
 
-// AddKey Adds the given key to the current user's authorised keys
-func AddKey(key, description string) error {
+// AddKey adds the given key to the current user's authorised keys
+func AddKey(key, description string, temp bool) error {
 	username, token, err := LoadToken(true)
-	address := fmt.Sprintf("%s/api/accounts/%s/keys", authhost, username)
-	// TODO: Check err and req.StatusCode
-	// TODO: clean up key struct
-	mkBody := func(key, description string) io.Reader {
-		pw := &struct {
-			Key         string `json:"key"`
-			Description string `json:"description"`
-		}{key, description}
-		b, _ := json.Marshal(pw)
-		return bytes.NewReader(b)
-	}
+	util.CheckErrorMsg(err, "This command requires login.")
```

Comment:
If I am not mistaken this function will never return errors (and rather die). 

I think we already discussed that the way we handle errors in the client is not really harmonized i.e. sometimes they peculate up and sometimes they don’t.  I guess its because we are in a transition period here. However, we should come up with a general solution. Shall we open an issue concerning libraries and stuff